dashboard/db/migrate/20170127183604_create_sign_ins.rb

```diff
@@ -0,0 +1,13 @@
+class CreateSignIns < ActiveRecord::Migration[5.0]
+  def change
+    create_table :sign_ins do |t|
```

Comment:
That is a very good question... Let me bubble that question up to others. My inclination is that we may not want to store it (for privacy reasons), though this should definitely be explicitly considered.